Sources/ScipioKit/Producer/PIF/PIFGenerator.swift

```diff
@@ -1,11 +1,12 @@
 import Foundation
 import TSCBasic
 import PIFKit
+import TSCUtility
 
 struct PIFGenerator {
     private let packageName: String
     private let packageLocator: any PackageLocator
-    private let toolchainLibDirectory: URL
+    private let toolchainLibDirectory: Foundation.URL
```

Comment:
Does URL conflicts?

Reply:
Yes, URL is also defined in TSCUtility, so we need to use the Foundation namespace explicitly.